src/Reflection/ReflectionParameter.php

```diff
@@ -208,8 +209,20 @@ private function parseDefaultValueNode() : void
         $defaultValueNode = $this->node->default;
 
         if ($defaultValueNode instanceof Node\Expr\ClassConstFetch) {
```

Comment:
After changes, there was only one condition added so I think it could remain inside `parseDefaultValueNode()`